.env.benefit-backend.example

```diff
@@ -0,0 +1,68 @@
+# Benefit backend
+DEBUG=1
+ENABLE_DEBUG_ENV=1
+APPLY_MIGRATIONS=1
+CREATE_SUPERUSER=1
+CORS_ALLOW_ALL_ORIGINS=1
+
+# Debugging options
+#
+# if changing DUMMY_COMPANY_FORM_CODE, also set NEXT_PUBLIC_MOCK_FLAG=1
+# DUMMY_COMPANY_FORM_CODE=16
+# NEXT_PUBLIC_MOCK_FLAG=1
+# DISABLE_AUTHENTICATION=1
+# DISABLE_TOS_APPROVAL_CHECK=1
+
+## Authentication
+DISABLE_AUTHENTICATION=0
+OIDC_RP_CLIENT_ID=
+OIDC_RP_CLIENT_SECRET=
+OIDC_OP_BASE_URL=https://tunnistus.test.hel.ninja/auth/realms/helsinki-tunnistus/protocol/openid-connect
+OIDC_OP_LOGOUT_CALLBACK_URL=https://localhost:8000/oidc/logout_callback/
+LOGIN_REDIRECT_URL=https://localhost:3000/
+LOGIN_REDIRECT_URL_FAILURE=https://localhost:3000/login?error=true
+LOGOUT_REDIRECT_URL=https://localhost:3000/login?logout=true
+
+
+EAUTHORIZATIONS_BASE_URL=https://asiointivaltuustarkastus.test.suomi.fi
+EAUTHORIZATIONS_CLIENT_ID=
+EAUTHORIZATIONS_CLIENT_SECRET=
+EAUTHORIZATIONS_API_OAUTH_SECRET=
+
+ADFS_LOGIN_REDIRECT_URL=https://localhost:8000/callback
+ADFS_TENANT_ID=
+ADFS_CLIENT_SECRET=
+ADFS_CLIENT_ID=
+ADFS_LOGIN_REDIRECT_URL_FAILURE=https://localhost:3100/login?error=true
+ADFS_CONTROLLER_GROUP_UUIDS=93e4f565-cde2-4b30-9226-006d2a508269,
```

Comment:
Is it ok to have a trailing comma here? Does it add an empty value to the list i.e. would the resulting list be ["93e4f565-cde2-4b30-9226-006d2a508269", ""]?

Reply:
It doesn't add an empty value, but I agree that it looks strange, fixed it